src/modules/verifiable-credentials/verifiable-credentials-base.service.ts

```diff
@@ -93,12 +101,43 @@ export abstract class VerifiableCredentialsServiceBase {
     proof_options: string
   ): Promise<string>;
 
-  constructor(protected readonly _signerService: SignerService) {}
+  constructor(
+    protected readonly _signerService: SignerService,
+    private readonly _storage: VCStorageClient
+  ) {}
+
+  /**
+   * @description requests verifiable credentials of type `type`. Only EWF credentials currently supported
+   * @returns credentials query with matching verifiable presentations
+   */
+  async initiateExchange({ type, url }: ExchangeInvitation) {
+    if (type !== EWF_EXCHANGE_INVITATION) {
+      throw new Error('Only EWF exchange is supported');
+    }
+
+    const { data: vpRequest } = await axios.post<VpRequest>(url);
```

Comment:
Should this type be `VpRequest` with capital `V`?
```suggestion
    const { data: VpRequest } = await axios.post<VpRequest>(url);
```

Reply:
It's not a type, it's an alias :face_in_clouds:

Reply:
Ah right! I didn't recognize it as that, thanks!